lib/autoload/kuroko2/workflow/task/auto_skip_error.rb

```diff
@@ -11,8 +11,7 @@ def execute
         end
 
         def auto_skip_error?
-          return @auto_skip_error if defined? @auto_skip_error
-          @auto_skip_error = option ? TRUE_OPTIONS.include?(option.strip) : false
+          @auto_skip_error ||= option ? TRUE_OPTIONS.include?(option.strip) : false
```

Comment:
`@auto_skip_error` was already memoized, but this change breaks the memoization.
When `option` or `TRUE_OPTIONS.include?(option.strip)` is evaluated to false, `@auto_skip_error ||= expr` is evaluated every time.